app/src/main/java/sw19/moring03/paint/tools/ChristmasTreeTool.java

```diff
@@ -0,0 +1,69 @@
+package sw19.moring03.paint.tools;
+
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Rect;
+
+import java.util.ArrayList;
+
+import sw19.moring03.paint.utils.PointF;
+
+public class ChristmasTreeTool extends Tools {
```

Comment:
to support the different colors (at least the ones who are directly selectable (12) - not by setting the RGB values) png for the different colors can be saved and then selected according to the chosen color.

Reply:
definitely not doing that one